server/service/src/requisition/response_requisition/update.rs

```diff
@@ -64,6 +65,21 @@ pub fn update_response_requisition(
             vec![Record::RequisitionRow(requisition.requisition_row.clone())],
         )
     );
+
+    if input.status == Some(UpdateResponseRequstionStatus::Finalised) {
```

Comment:
In theory, some aspects of response requisitions and invoices are updatable even after being finalised. In mSupply a comment can be edited after finalisation - in which case we would have log entries saying that the status has changed to finalised after each edit. Even when already finalised.

Is it much trouble to check the current status and only add this log entry if the status changes to finalised?

Not a big issue currently, because the front end blocks editing after finalising, but I would expect that editing of comments will be requested at some point.